src/App.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import Card from './Card';
+
+function App() {
+  
```

Comment:
One thing to help render out the cards - you can store an array with the user IDs, and then map over it to render out each card. It can make things a bit easier to maintain. 

```jsx
const userIds = [1, 2, 3, 4, 5, 6]; 

return (
  <div className="App">
        { userIds.map( (userId) => <Card key={userId} userId={userId} /> }
  </div>
);